solana/program_v2/packages/client/cli/src/commands/gatekeeper/setstate.ts

```diff
@@ -79,7 +85,7 @@ export default class SetState extends Command {
     const initialState = gatekeeperAccount?.state as GatekeeperState;
 
     const stateChangeSignature = await networkService
-      .setGatekeeperState(targetState)
+      .setGatekeeperState(targetState, undefined, network)
```

Comment:
i don't feel this should be needed to be explicitly set... maybe a problem deeper in the code then?

Reply:
by "this" i mean `undefined, network`

Reply:
The network is a required parameter but it doesn't have a default value, so you need to pass in undefined for the authority first before you can give the network. Probably should set it so that network is first as the only param without a default value

Reply:
Fixed